app/src/main/java/com/celzero/bravedns/database/AppDatabase.kt

```diff
@@ -378,6 +378,16 @@ abstract class AppDatabase : RoomDatabase() {
                         execSQL(
                             "INSERT OR REPLACE INTO DoHEndpoint(dohName,dohURL,dohExplanation, isSelected,isCustom,modifiedDataTime,latency) values('Quad9 Secure','https://dns.quad9.net/dns-query','Quad9 routes your DNS queries through a secure network of servers around the globe.',0,0,0,0)"
                         )
+                        execSQL(
+                            "INSERT OR REPLACE INTO DoHEndpoint(dohName,dohURL,dohExplanation, isSelected,isCustom,modifiedDataTime,latency) values('Mullvad DNS','https://dns.mullvad.net/dns-query','Mullvad's DNS server is a no-logging service, usable even if you're not a Mullvad customer.',0,0,0,0)"
```

Comment:
> usable even if you're not a Mullvad customer.

No need to mention that to RDNS users. Also using "_Mullvad's DNS_" keyword is redundant as it's already mentioned in the title.
I think better would be something like "Audited DNS server with no-logging of user activity policy."